src/main/resources/db/changelog/changes/000_initial_schema.yaml

```diff
@@ -6,21 +6,21 @@ databaseChangeLog:
     - createTable:
         columns:
         - column:
-            autoIncrement: true
+            autoIncrement: false
```

Comment:
I think that we have to create a new file. We should recover the `src/main/resources/db/changelog/changes/001_add_team_name.yaml` file, that has been deleted, and create a new `002_...` with all the changes.

Here's the docs about it (see the YAML section): https://docs.liquibase.com/change-types/modify-data-type.html

Reply:
Oh, right, i think i did it the wrong way as i recreated the entire DB resetting the databasechangelog table and re-executing the modified initial script